src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -808,8 +808,9 @@ public void onMaximizeConsole()
    {
       ArrayList<LogicalWindow> results = new ArrayList<>();
 
+      // Do not include hiddenTabSet in panes since it should never be displayed
       JsArrayString panes = config.getQuadrants();
-      for (int i = 0; i < panes.length(); i++)
+      for (int i = 0; i < 4; i++)
```

Comment:
It will be very easy for this to break silently if the `panes` array length changes in the future (which of course it could). Maybe we should just `continue` through the loop for this tabset?

Reply:
Good plan, I've amended the commit with the change.